go.mod

```diff
@@ -30,7 +31,7 @@ require (
 	github.com/jonboulle/clockwork v0.1.0 // indirect
 	github.com/juju/ratelimit v1.0.1
 	github.com/kylelemons/godebug v1.1.0
-	github.com/letsencrypt/pkcs11key v2.0.0+incompatible // indirect
+	github.com/letsencrypt/pkcs11key v2.0.1-0.20170608213348-396559074696+incompatible // indirect
```

Comment:
Why can't we use the released version (v2.0.0)? Also, should we let them know that Go is flagging their repo as "incompatible"?

Reply:
The Trilian repo uses `config.PublicKeyPath` which is not present in the latest release.

Reply:
The reason it is being marked as "incompatible" is because they have a release version > 1 that does not have the version in the package path, as recommended in go modules.

Reply:
1) We use features that aren't in the last release, hence the need to use HEAD
2) It would be nice if they turned on Go modules for their repo so we know what set of deps they expect are
3) It's marked as incompatible because v2 is not in the path

Reply:
Could you open an issue on their repo basically just restating this please?

Reply:
https://github.com/letsencrypt/pkcs11key/issues/21
https://github.com/letsencrypt/pkcs11key/issues/22